src/lwc/geFormRenderer/elevateCaptureGroup.js

```diff
@@ -5,35 +5,28 @@ class ElevateCaptureGroup {
 
     constructor(elevateBatchId) {
         this.elevateBatchId = elevateBatchId;
-        this._hasAddRun = false;
     }
 
     async add(tokenizedGift) {
-        if (!this.elevateBatchId) {
-            this.elevateBatchId = await this.create();
-        }
-
         try {
+            if (!this.elevateBatchId) {
```

Comment:
Related to what I mentioned above, is this conditional check necessary moving forward? We shouldn't expect a batch Id to be present in at this point in the flow, especially now that we're removing the add to batch "retry" functionality from this method.

Reply:
Updated!